nautobot/ipam/factory.py

```diff
@@ -20,12 +30,26 @@ class Meta:
     is_private = factory.LazyAttribute(lambda rir: rir.name.startswith("RFC"))
 
     has_description = factory.Faker("pybool")
-    description = factory.Maybe("has_description", factory.Faker("sentence"), "")
+    description = factory.Maybe("has_description", factory.Faker("text", max_nb_chars=200), "")
 
-    # TODO custom field data?
 
+class AggregateFactory(PrimaryModelFactory):
+    """
+    Create random aggregates and 50% of the time generate prefixes within the aggregate IP space. Child prefixes
+    create nested child prefixes and ip addresses within the prefix IP space. Defaults to creating 0-4 child prefixes
+    which generate 0-4 grandchildren. Set child_prefixes__max_count to an integer when calling the factory creation
+    methods (`create()`, `create_batch()`, etc) to override the maximum number of child prefixes generated. Set
+    child_prefixes__children__max_count to an integer when calling the factory creation methods (`create()`, `batch_create()`, etc)
+    to override the maximum number of grandchildren generated.
+
+    Examples:
+    AggregateFactory.create_batch(20) # create 20 aggregates, approximately half will generate 0-4 child prefixes
+                                      # which will create child prefixes and ip addresses
+    AggregateFactory.create_batch(child_prefixes__max_count=0) # create 20 aggregates with no child prefixes
+    AggregateFactory.create_batch(child_prefixes__children__max_count=0) # create 20 aggregates, approximately
+                                      # half will generate 0-4 child prefixes that will not create any children
```

Comment:
This reads a little funky. Could we instead have each call on a line of its own with the text preceding it instead of being inline as a comment?

Something like this:


```suggestion

    Create 20 Aggregates, approximately half will generate 0-4 child prefixes which will create child prefixes and ip addresses:
    
        >>> AggregateFactory.create_batch(20)
    
    Create 20 Aggregates with no child prefixes:
    
        >>> AggregateFactory.create_batch(child_prefixes__max_count=0)
    
    Create 20 Aggregates, approximately half will generate 0-4 child prefixes that will not create any children:
    
        >>> AggregateFactory.create_batch(child_prefixes__children__max_count=0) 
```